docs/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+group = rootProject.group
+version = rootProject.version
+
+plugins {
+    id("ai.kotlin.jvm")
+    alias(libs.plugins.kotlin.serialization)
+    alias(libs.plugins.knit)
+}
+
+dependencies {
+    api(project(":agents:agents-test"))
+    api(project(":koog-agents"))
```

Comment:
minor: api/implementation does not make any sense in `docs` since `docs` are not imported anywhere